src/Courses/CourseInterfaces.tsx

```diff
@@ -50,9 +55,11 @@ export class UserObject {
     firstName?: string;
     lastName?: string;
     id: number = -1;
+    name: string;
 
     public constructor(init?:Partial<UserObject>) {
         Object.assign(this, init);
+        this.name = `${this.firstName} ${this.lastName}`;
```

Comment:
Should we make this a getter instead? The code that calls it will remain the same but if the values update the getter will be correct